site/docs/uk/advanced/business.md

```diff
@@ -0,0 +1,141 @@
+Загалом, це корисно, якщо ви ведете бізнес у Telegram, а інший користувач є вашим клієнтом.
+
+> Якщо ви ще не знайомі з Telegram Business, ознайомтеся з [офіційною документацією](https://core.telegram.org/bots#manage-your-business) від Telegram, перш ніж продовжити.
+
+Звісно, grammY повністю підтримує це.
+
+## Обробка бізнес-повідомлень
+
+Бот може керувати приватним чатом між двома користувачами через Telegram Business --- акаунт, який підписаний на бізнес-підписку Telegram.
+Керування приватними чатами здійснюється через обʼєкт _business connection_, який виглядає [ось так](/ref/types/businessconnection).
+
+### Отримання бізнес-повідомлень
+
+Після встановлення бізнес-підключення бот буде **отримувати повідомлення** від _обох учасників чату_.
+
+```ts
+bot.on("business_message", async (ctx) => {
+  // Отримуємо обʼєкт повідомлення.
+  const message = ctx.businessMessage;
+  // Скорочені методи працюють, як і очікувалося.
+  const msg = ctx.msg;
+});
+```
+
+Наразі незрозуміло, хто з двох учасників чату надіслав повідомлення.
+Це може бути повідомлення вашого клієнта, а може бути повідомлення, надіслане особисто вами (не вашим ботом)!
+
+Отже, нам потрібно розрізнити цих двох користувачів.
+Для цього нам потрібно перевірити вищезгаданий обʼєкт бізнес-підключення.
+Бізнес-підключення повідомляє нам, хто є користувачем бізнес-акаунта, тобто ідентифікатор користувача вас або одного з ваших співробітників.
+
+```ts
+bot.on("business_message", async (ctx) => {
+  // Отримуємо інформацію про бізнес-підключення.
+  const conn = await ctx.getBusinessConnection();
+  const employee = conn.user;
+  // Перевіряємо, хто надіслав це повідомлення.
+  if (ctx.from.id === employee.id) {
+    // Це повідомлення надіслали ви.
+  } else {
+    // Це повідомлення надіслав ваш клієнт.
+  }
+});
+```
+
+Ви також можете пропустити виклик `getBusinessConnection` для кожного оновлення, виконавши [це](#робота-з-бізнес-підключеннями).
+
+### Надсилання повідомлень
+
+Ваш бот може **надсилати повідомлення** до цього чату _не будучи учасником чату_.
+Це працює, як і очікувалося, за допомогою `ctx.reply` і всіх його варіантів.
+grammY перевіряє, чи доступний [скорочений метод контексту](../guide/context#скорочені-методи) `ctx.businessConnectionId`, щоб надіслати повідомлення до керованого бізнес-чату.
+
+```ts
+bot.on("business_message").filter(
+  async (ctx) => {
+    const conn = await ctx.getBusinessConnection();
+    return ctx.from.id !== conn.user.id;
+  },
+  async (ctx) => {
+    // Автоматично відповідаємо на всі запитання клієнтів.
+    if (ctx.msg.text.endsWith("?")) {
+      await ctx.reply("Скоро.");
+    }
+  },
+);
+```
+
+Це виглядатиме так, ніби ви відправили повідомлення особисто.
+Ваш клієнт не зможе визначити, чи було повідомлення надіслано вручну, чи через бота.
+Однак ви побачите невеликий індикатор щодо цього.
+Але ваш бот, ймовірно, відповідає набагато швидше, ніж ви.
+Перепрошуємо.
+
+## Просування
```

Comment:
I can't find a suitable translation for "Going further". @deptyped @LWJerri, if you have any ideas, please leave them in this comment

Reply:
Just translate something else. Here are a few options:
- Next steps
- Advanced features
- How to continue
- Going beyond the basics
- Whatever floats your goat

Reply:
```suggestion
## Просунуті можливості
```